pxr/usd/usdLux/lightAPI.h

```diff
@@ -46,6 +46,40 @@ class SdfAssetPath;
 /// type (e.g. RectLight or DistantLight) or is applied directly to a Gprim 
 /// that should be treated as a light.
 /// 
+/// <b>Quantities and Units</b>
+/// 
+/// Most renderers consuming OpenUSD today are RGB renderers, rather than
+/// spectral. Units in RGB renderers are tricky to define as each of the red,
+/// green and blue channels transported by the renderer represents the
+/// convolution of a spectral exposure distribution, e.g. CIE Illuminant D65,
+/// with a sensor response function, e.g. CIE 1931 𝓍̅. Thus the main quantity
+/// in an RGB renderer is neither radiance nor luminance, but "integrated
+/// radiance" or "tristimulus weight".
+/// 
+/// The emision of a default light with `intensity` 1 and `color` [1, 1, 1] is
+/// an Illuminant D spectral power distribution with chromaticity matching the
+/// rendering color space white point, normalized such that a ray normally
+/// incident upon the sensor with EV0 exposure settings will generate a pixel
+/// value of [1, 1, 1] in the rendering color space.
+/// 
+/// Given the above definition, that means that the luminance of said default
+/// light will be 1 *nit (cd∕m²)* and its emission spectral radiance
+/// distribution is easily computed by appropriate normalization.
+/// 
+/// For brevity, the term *emission* will be used in the documentation to mean
+/// "emitted spectral radiance" or "emitted integrated radiance/tristimulus
+/// weight", as appropriate.
+/// 
+/// Note that some color spaces, most notably ACES, define their white points
```

Comment:
I do wonder if we should reword this section about ACES. I'm thinking that we should convey this information that because a standard uplift is not specified beyond the requirement of a round-trip, we recommend using color spaces specified with respect to a standard illuminant.

The intent of this suggestion is to encourage the use of color spaces that are naturally conducive to physical colorimetry. If we word this section as it is now, it seems to invite more troubleshooting and a continuation of work arounds and hacks to accommodate adaptation and "uplift" which leaves us outside the realm of strictly measurable science.

Reply:
@anderslanglands @meshula - does this seem acceptable?

https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/a1d7d8328149bf6067832d65c79f003fcfc262fa#diff-11634bb5438749e3c529c9f44011d464e79bdb1352c0b7fc7c9408bece4415e4R73-R82

Reply:
If we’re going to do this then do we want to say something like “encourage the use of colour spaces whose white point has a well-defined spectral representation, such as D65”?

Reply:
Tweaked: https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/e493051f2d48654482586cbc32d46eb3a84b930d#diff-ec32e12ccc5be018c3a056d787ed4acdddd0653b33d1cc95593a8e5c26a1a75fR51

Reply:
I like this latest version the best. I just added new notes about photometry above, and the new formulation works with radiometry.